client/packages/android/app/src/main/assets/capacitor.config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"appId": "org.openmsupply.client",
-	"appName": "openmsupply-client",
+	"appName": "Open mSupply",
```

Comment:
it was the change in stirngs.xml which made it work, this was done for completeness